src/renderer/utils/markdown.ts

```diff
@@ -130,6 +130,60 @@ const Markdown = {
 
     },
 
+    // Wikilink
+    wikilink () {
+
+      const {path: notesPath} = Config.notes;
+
+      if ( !notesPath ) return [];
+
+      var matches = [];
+      return [
+        { // Link
+          type: 'lang',
+          regex: /(?<!```(?: *)(?:[^\s`~]*)\n(?:[\s\S]*?)|`)\[{2}(.*?\|?.*?)\]{2}/g,
+          replace (match, $1, $2) {
+            matches.push($1);
+            var n = matches.length - 1;
+            return '%PLACEHOLDER' + n + '%';
+          }
+        },
+        {
+          type: 'output',
+          filter (text) {
+                for (var i=0; i< matches.length; ++i) {
+                    var content = matches[i]
+
+                    var splits = content.split("|"); 
+
+                    if (splits.length === 1) {
+                      var wikiLink = splits[0].trim();
+                      var linkText = splits[0].trim();
+                    }
+                    else {
+                      var wikiLink = splits[1].trim();
+                      var linkText = splits[0].trim();
+                    };
+
+                    wikiLink = decodeURI ( wikiLink );
+                    var basename = [wikiLink, 'md'].join('.'); // the extension should be configurable (= default extension)
```

Comment:
The extension should be optional, you should check if one has already been provided before appending `.md`.

Reply:
fixed